### ISSUE TITLE
[Programmatic Access - Make code - Home Page]: Aria expanded attribute is not defined for the 'Plant watering' button.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#6085]
### User Experience:
Visually impaired people who depend on screen reader will get impacted if aria-expanded attribute is not defined for the button. Due to which the end users will not be able to know the current state of the button and cannot access it efficiently.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: [https://makecode.microbit.org/beta#](https://makecode.microbit.org/beta#)
2. Home page will open.
3. Navigate to 'Science' section and observe whether aria-expanded attribute is defined for the 'Plant watering' button or not.

### Actual Result:

**Issue:** Aria expanded attribute is not defined for the 'Plant watering' button.

**Observation:**

Keyboard focus lands on the revealed content by expanding the 'Plant watering' button.

### Expected Result:
Aria expanded attribute should be defined for the 'Plant watering' button.

**Example:** Aria-expanded="true" should be defined when it is in expanded state and aria-expanded="false" should be defined when it is in collapsed state.

**Note:** Same issue is observed for the similar buttons present on the page.

### MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
Yes (Refer attachment)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- **OS Version:** Windows 11 24H2 (OS Build:26100.3037)
- **Browser**: Chrome Version 132.0.6834.196 (Official Build) (64-bit)
- **Test Matrix:** Chrome + NVDA
- **URL:** [https://makecode.microbit.org/.](https://makecode.microbit.org/beta#)

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/d3f44928-d5a9-4f7f-a2f5-557c6b525993" />

**Comment by vroopea** [comment]
#Accessibility; #A11yMAS; #A11ySev3; #MAS4.1.2; #AILimited; #HCL; #Benchmark; #Win11-Chrome; #BM-HCL-MakeCode-Feb2025

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug
